n3c_owl_ingest/n3c_owl_ingest.py

```diff
@@ -273,18 +331,52 @@ def run_ingest(split_by_vocab: bool = False, skip_if_in_release: bool = True, me
                 continue
             # noinspection PyBroadException
             try:
-                via_robot_template(concept_subclass_ofs, num_subclass_cols, group, outpath)
+                via_robot_template(group_df, rel_maps, outpath, use_cache=use_cache)
             except Exception:
                 os.remove(outpath)
             t_i2 = datetime.now()
             print(f'Vocab {name} finished in {(t_i2 - t_i1).seconds} seconds')
             i += 1
     else:
-        via_robot_template(concept_subclass_ofs, num_subclass_cols, concept_df, outpath)
+        via_robot_template(concept_df, rel_maps, outpath, use_cache=use_cache)
+
+
+def cli():
```

Comment:
Added a CLI. RxNorm is one of the options, as there is special logic for that one.